core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/kubernetes/upgrade-masterN.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: upgrade-master | Drain master in preparation for maintenance
+  include_tasks: utils/drain.yml
+
+- name: upgrade-master | Upgrade, configure packages
+  block:
+    - name: upgrade-master | Hold packages for Debian family
+      include_tasks: Debian/hold-packages.yml
+      when: ansible_os_family == "Debian"
+
+    - name: upgrade-master | Install kubeadm
+      include_tasks: "{{ ansible_os_family }}/install-kubeadm.yml"
+
+    - name: upgrade-master | Upgrade master {{ inventory_hostname }}
+      command: >-
+        kubeadm upgrade node
+      register: result
+      until:
+        - result is succeeded
+      retries: 20
+      delay: 30
+      changed_when: false
+
+    - name: Install kubelet and kubectl for {{ version }}
+      include_tasks: >-
+        {%- if cni_in_kubelet is undefined or not cni_in_kubelet -%}
+          {{ ansible_os_family }}/install-packages.yml
+        {%- else -%}
+          {{ ansible_os_family }}/install-packages-cni-in-kubelet.yml
+        {%- endif -%}
+      when:
+        - result is succeeded
+
+- name: upgrade-master | Wait for cluster's readiness
+  include_tasks: utils/wait.yml
+
+- name: upgrade-master | Upgrade kubeadm-config.yml if exists
+  include_tasks: upgrade-kubeadm-config.yml
+
+- name: upgrade-master | Upgrade Docker  # this may restart Docker daemon
+  include_tasks: docker.yml
+
+- name: upgrade-master | Reload kubelet and docker
+  include_tasks: utils/reload-kubelet-and-docker.yml
+
+- name: upgrade-master | Uncordon master - mark master as schedulable
+  include_tasks: utils/uncordon.yml
```

Comment:
```suggestion
---
- name: upgrade-masterN | Drain master in preparation for maintenance
  include_tasks: utils/drain.yml

- name: Upgrade, configure packages
  block:
    - name: upgrade-masterN | Hold packages for Debian family
      include_tasks: Debian/hold-packages.yml
      when: ansible_os_family == "Debian"

    - name: upgrade-masterN | Install kubeadm
      include_tasks: "{{ ansible_os_family }}/install-kubeadm.yml"

    - name: upgrade-masterN | Upgrade master {{ inventory_hostname }}
      command: >-
        kubeadm upgrade node
      register: result
      until:
        - result is succeeded
      retries: 20
      delay: 30
      changed_when: false

    - name: name: upgrade-masterN | Install kubelet and kubectl for {{ version }}
      include_tasks: >-
        {%- if cni_in_kubelet is undefined or not cni_in_kubelet -%}
          {{ ansible_os_family }}/install-packages.yml
        {%- else -%}
          {{ ansible_os_family }}/install-packages-cni-in-kubelet.yml
        {%- endif -%}
      when:
        - result is succeeded

- name: upgrade-masterN | Wait for cluster's readiness
  include_tasks: utils/wait.yml

- name: upgrade-masterN | Upgrade kubeadm-config.yml if exists
  include_tasks: upgrade-kubeadm-config.yml

- name: upgrade-masterN | Upgrade Docker  # this may restart Docker daemon
  include_tasks: docker.yml

- name: upgrade-masterN | Reload kubelet and docker
  include_tasks: utils/reload-kubelet-and-docker.yml

- name: upgrade-masterN | Uncordon master - mark master as schedulable
  include_tasks: utils/uncordon.yml

- name: upgrade-masterN | Verify component versions and node status
  include_tasks: kubernetes/verify-upgrade.yml
```